src/main/java/frc/robot/subsystems/mailbox/MailboxBelts.java

```diff
@@ -18,21 +18,29 @@
 
 /** Subsystem for the mailbox belt. */
 public class MailboxBelts extends SubsystemBase {
-  private CANSparkMax beltMotor;
+  private CANSparkMax upperBeltMotor;
+  private CANSparkMax lowerBeltMotor;
 
   /** Constructer for MailboxBelts subsystem */
   public MailboxBelts() {
-    beltMotor = new CANSparkMax(Constants.MailboxBelts.BELT_MOTOR_ID, MotorType.kBrushless);
+    upperBeltMotor =
+        new CANSparkMax(Constants.MailboxBelts.UPPER_BELT_MOTOR_ID, MotorType.kBrushless);
+    lowerBeltMotor =
+        new CANSparkMax(Constants.MailboxBelts.LOWER_BELT_MOTOR_ID, MotorType.kBrushless);
+
+    upperBeltMotor.setInverted(Constants.MailboxBelts.UPPER_BELT_MOTOR_INVERTED);
+    lowerBeltMotor.setInverted(Constants.MailboxBelts.LOWER_BELT_MOTOR_INVERTED);
   }
 
   /** Runs the belt. */
   public void runBelts() {
-    beltMotor.set(Constants.MailboxBelts.BELT_MOTOR_SPEED);
+    upperBeltMotor.set(Constants.MailboxBelts.BELT_MOTOR_SPEED);
+    lowerBeltMotor.follow(upperBeltMotor, Constants.MailboxBelts.BELTS_FOLLOWER_INVERSE_STATE);
```

Comment:
DO NOT run the follow call in a method that will be run by `execute()`, ESPECIALLY not after you call `.set(double)` on the motor it's following. `.follow(CANSparkMax, boolean)` changes a setting on the motor controller that tells it to listen to the CAN commands for the other motor, so you want to call it when you initialize the motors.